applications/GeoMechanicsApplication/tests/test_dsettlement_validation.py

```diff
@@ -1124,10 +1125,10 @@ def test_settlement_fully_saturated_column_low_permeability(self):
 
         # Check some results
         actual_settlement_after_one_hundred_days = reader.nodal_values_at_time(
-            "TOTAL_DISPLACEMENT", 8640000, output_stage_2, [104]
+            "TOTAL_DISPLACEMENT", 8640000, output_stage_3, [104]
```

Comment:
Perhaps we can write the intended time more clearly by using the unit conversion function:
```suggestion
            "TOTAL_DISPLACEMENT", unit_conversions.days_to_seconds(100), output_stage_3, [104]
```

Reply:
Adapted.